src/Nordea/Components/Modal.elm

```diff
@@ -68,6 +70,10 @@ view config attrs children =
                         , flexDirection column
                         , minWidth (rem 18)
                         , maxWidth (rem 60)
+                        , margin (auto)
+                        , Media.withMedia
+                            [ Media.only Media.screen [ Media.maxWidth (rem 46.875) ] ]
```

Comment:
I think we should try to use minWidth instead of maxWidth, because that follows the styling prinsipp of mobile first. So style for mobile, and change styling for bigger screens when needed.

Reply:
And - is it also possible to use 47 or 46 rem instead of 46.875? 😊